objects.go

```diff
@@ -63,11 +97,130 @@ type Network struct {
+	PLATFORM                 string `json:"platform,omitempty"`
+	ServiceTypeConfiguration string `json:"service_type_configuration,omitempty"`
+}
+
+// License represents license wapi object
+type License struct {
+	IBBase           `json:"-"`
+	Ref              string `json:"_ref,omitempty"`
+	ExpirationStatus string `json:"expiration_status,omitempty"`
+	ExpiryDate       int    `json:"expiry_date,omitempty"`
+	HwID             string `json:"hwid,omitempty"`
+	Key              string `json:"key,omitempty"`
+	Kind             string `json:"kind,omitempty"`
+	Limit            string `json:"limit,omitempty"`
+	LimitContext     string `json:"limit_context,omitempty"`
+	Licensetype      string `json:"type,omitempty"`
+}
+
+type MemberResult struct {
+	Ref      string     `json:"_ref,omitempty"`
+	HostName string     `json:"host_name,omitempty"`
+	Nodeinfo []NodeInfo `json:"node_info,omitempty"`
+	TimeZone string     `json:"time_zone"`
+}
+
+// CapacityReport represents capacityreport object
+type CapacityReport struct {
+	IBBase `json:"-"`
+	Ref    string `json:"_ref,omitempty"`
+
+	Name         string                   `json:"name,omitempty"`
+	HardwareType string                   `json:"hardware_type,omitempty"`
+	MaxCapacity  int                      `json:"max_capacity,omitempty"`
+	ObjectCount  []map[string]interface{} `json:"object_counts,omitempty"`
+	PercentUsed  int                      `json:"percent_used,omitempty"`
+	Role         string                   `json:"role,omitempty"`
+	TotalObjects int                      `json:"total_objects,omitempty"`
+}
+type NTPSetting struct {
+	enable_ntp bool                   `json:"enable_ntp,omitempty"`
+	NTPAcl     map[string]interface{} `json:"ntp_acl,omitempty"`
+	NTPKeys    []string               `json:"ntp_keys,omitempty"`
+	NTPKod     bool                   `json:"ntp_kod,omitempty"`
+	NTPServers []string               `json:"ntp_servers,omitempty"`
+}
+
+type Grid struct {
+	IBBase `json:"-"`
+	Ref    string `json:"_ref,omitempty"`
+}
+
+type GridResult struct {
+	Ref        string     `json:"_ref,omitempty"`
+	Name       string     `json:"name,omitempty"`
+	NTPSetting NTPSetting `json:"ntp_setting"`
+}
+
+func NewGrid(grid Grid, returnFields []string) *Grid {
+	result := grid
+	result.objectType = "grid"
+	result.returnFields = returnFields
+	return &result
+}
+
+func NewLicense(license License, returnFields []string) *License {
+	result := license
+	result.objectType = "member:license"
+	result.returnFields = returnFields
+	return &result
+}
+
+func NewCapcityReport(capReport CapacityReport, returnFields []string) *CapacityReport {
+	res := capReport
+	res.objectType = "capacityreport"
+	res.returnFields = returnFields
+	return &res
+}
+
+func NewMember(member Member, returnFields []string) *Member {
+	res := member
+	res.objectType = "member"
+	res.returnFields = returnFields
+	return &res
+}
+
 func NewNetwork(nw Network) *Network {
```

Comment:
Consistency...

Originally, the NewXYZ() method is defined right after the definition of the XYZ object.
I have no objection if you like to change it to have all object definitions followed by method definitions.

BUT, please only do it one way.

Reply:
Done, ill keep it as you mentioned. change it through out to maintain consistency